### PR TITLE
funny

### PR DESCRIPTION
**Pull request by Bulvius1**

### CODE REVIEW

File: src/commands/Funny [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+import { CustomEmbed, ErrorEmbed } from "../models/EmbedBuilders";
+import { addProgress } from "../utils/functions/economy/achievements.js";
+import { createUser, userExists } from "../utils/functions/economy/utils.js";
+import { getXp, updateXp } from "../utils/functions/economy/xp.js";
+import { getMember } from "../utils/functions/member.js";
+import { addCooldown, getResponse, onCooldown } from "../utils/handlers/cooldownhandler.js";
+
+const cache = new Map<string, number>();
+
+const cmd = new Command("funny", "measure how much funny u are", "fun").setAliases([
```
Comment (tekoh) [inline src/commands/Funny:18]: fix grammar

File: src/commands/Funny [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+  "howfunny",
+  "unfunny",
+]);
+
+cmd.slashEnabled = true;
+cmd.slashData.addUserOption((option) =>
```
Comment (tekoh) [inline src/commands/Funny:24]: no slash command needed - remove

File: src/commands/Funny [hunk 3]
```diff
@@ -0,0 +1,150 @@
...
+    funnyText = "Even the google cant help you";
+  } else if (funnyAmount >= 15) {
+    funnyEmoji = "🤨🤨";
+    funnyText = "Your mama jokes are funnier than u";
+  } else if (funnyAmount >= 7) {
+    funnyEmoji = "🤨⁉";
+    funnyText = "nobody is laughting.";
+  } else {
+    funnyEmoji = "🙄";
+    funnyText = "you're unfunny, but have 1 xp";
```
Comment (tekoh) [inline src/commands/Funny:126]: grammar

File: src/commands/funny
Comment (tekoh) [inline src/commands/funny:1]: why create two files?? delete this one and make the other one lowercase + it needs file extension

**Review by tekoh — CHANGES_REQUESTED** [review]

**Review by tekoh — CHANGES_REQUESTED** [review]
you still havent made the changes i requested. why did you request a review?

File: src/commands/Funny (hunk 1 above)
Comment (Bulvius1) [reply src/commands/Funny:18]: Fixed

File: src/commands/Funny (hunk 3 above)
Comment (Bulvius1) [reply src/commands/Funny:126]: fixed

File: src/commands/funny
Comment (Bulvius1) [reply src/commands/funny:1]: ok sorry

File: src/commands/Funny (hunk 2 above)
Comment (Bulvius1) [reply src/commands/Funny:24]: i hope this works

**Review by tekoh — CHANGES_REQUESTED** [review]
still broken

**Comment by tekoh** [comment]
closing bc unresponsive

**Pull request closed without merging**